### PR TITLE
Create example for POST check.

### PR DESCRIPTION
**Pull request by slz250**

### CODE REVIEW

File: monitoring/api/v3/uptime-check-client/snippets.py [hunk 1]
```diff
@@ -23,10 +23,10 @@
 
 
 # [START monitoring_uptime_check_create]
-def create_uptime_check_config(project_name, host_name=None,
```
Comment (busunkim96) [inline monitoring/api/v3/uptime-check-client/snippets.py:26]: @GoogleCloudPlatform/python-samples-owners Is it preferred to have one function per region tag, or is doubling up OK?

Comment (tmatsuo) [reply monitoring/api/v3/uptime-check-client/snippets.py:26]: Personally I'm ok with either way.

File: monitoring/api/v3/uptime-check-client/snippets_test.py [hunk 2]
```diff
@@ -36,15 +36,19 @@ def __init__(self):
         self.project_name = snippets.project_name()
 
     def __enter__(self):
-        # Create an uptime check config.
-        self.config = snippets.create_uptime_check_config(
+        print('test');
```
Comment (tmatsuo) [inline monitoring/api/v3/uptime-check-client/snippets_test.py:39]: Remove?

**Comment by slz250** [comment]
@tmatsuo @busunkim96  

Is there a linter I can run this PR against to fix the Kokoro CI - Lint error?

**Comment by busunkim96** [comment]
@simonz130 `black` should fix those lint problems:

`python3 -m pip install black` and `black snippets.py snippets_test.py`

**Comment by tmatsuo** [comment]
@slz250

If you have Docker installed and it's runnable by normal user, try this command:

```
$ cd monitoring/api/v3/uptime-check-client
$ TRAMPOLINE_DOCKERFILE=none ../../../../scripts/run_tests_local.sh . lint
```

**Pull request merged**